keyform/templates/keyform/emails/status_update.html

```diff
@@ -0,0 +1,422 @@
+<!doctype html>
```

Comment:
If you snagged this template from [here](https://github.com/leemunroe/responsive-html-email-template/blob/master/email.html) we should give them credit.